src/entities/Ball/Ball.ts

```diff
@@ -1,17 +1,23 @@
 import Phaser from 'phaser';
+import {
+  ballBounce,
+  ballFriction,
+  ballRadius,
+} from '@src/entities/Ball/sliderInputs';
 
 const BALL_COLOR = 0x1f4f8b;
 
 export default class Ball extends Phaser.GameObjects.Arc {
   element: Phaser.Physics.Matter.Sprite;
 
   constructor(scene: Phaser.Scene) {
```

Comment:
I'd prefer if the ball values were parameters of the constructor to keep the logik of getting the values seperate from the Ball class

Reply:
ah yeah makes sense, I will split that up

Reply:
In this issue?

Reply:
yeah will get onto it in this issue

was awaiting a response for the v-model suggestion which might have turned the whole implementation on its head